gorfc/gorfc.go

```diff
@@ -15,8 +15,7 @@ package gorfc
 #cgo linux CFLAGS: -DSAPwithUNICODE -D__NO_MATH_INLINES -DSAPwithTHREADS -DSAPonLIN
 #cgo linux CFLAGS: -O2 -minline-all-stringops -g -fno-strict-aliasing -fno-omit-frame-pointer
 #cgo linux CFLAGS: -m64 -fexceptions -funsigned-char -Wall -Wno-uninitialized -Wno-long-long
-#cgo linux CFLAGS: -Wcast-align -pthread -pipe
-#cgo linux CFLAGS: -Wunused-variable
+#cgo linux CFLAGS: -Wcast-align -pthread -pipe -Wno-unused-variable
```

Comment:
die beiden Zeilen in eine zusammengefasst und die Flag unused-variable gegen no-unused-variable ersetzt um Probleme mit ungenutzten Variablen im C-Code zu ignorieren